pkg/feeds/npm/npm.go

```diff
@@ -179,11 +185,14 @@ func fetchAllPackages(registryURL string) ([]*feeds.Package, []error) {
 		uniquePackages[pkg.Title]++
 	}
 
+	n := 0
 	for pkgTitle, count := range uniquePackages {
-		go func(pkgTitle string, count int) {
-			// Before requesting, wait, so all the requests don't arrive at once.
-			jitter := time.Duration(rand.Intn(maxJitterMillis)) * time.Millisecond //nolint:gosec
-			time.Sleep(jitter)
+		go func(id int, pkgTitle string, count int) {
+			if id > minJitterThreshold {
```

Comment:
...and this by `if applyJitter`? 

I think the intent would be clearer this way